integration_tests/pages/barcode/chooseBarcodeOption.ts

```diff
@@ -34,4 +36,10 @@ export default class ChooseBarcodeOptionPage extends Page {
   hasErrorContaining = (partialMessage: string): void => {
     cy.get('.govuk-error-summary__list').should('contain', partialMessage)
   }
+
+  static goToPage = (): ChooseBarcodeOptionPage => ReviewRecipientsPage.goToPage().happyPath()
+
+  happyPathImage = (): GenerateBarcodeImagePage => this.continueToImage()
+
+  happyPathCoversheet = (): SelectEnvelopeSizePage => this.continueToCoversheet()
```

Comment:
Some pages will need multiple `happyPath...` methods to model the various good ways through the page.